storyblok/components.json

```diff
@@ -78,7 +78,9 @@
       "all_presets": [],
       "preset_id": null,
       "real_name": "accordion_block_brand_pivot",
-      "component_group_uuid": null
+      "component_group_uuid": null,
+      "color": null,
+      "icon": null
```

Comment:
These are new fields from Storyblok where you can add some options to the admin view